### PR TITLE
Check index.html

### PR DESCRIPTION
**Pull request by jlmonroy13**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ parameters:
     default: ""
   REPOSITORY:
     type: string
-    default: "https://github.com/thales-markets/thales-sport-markets"
+    default: "https://github.com/jlmonroy13/vite-project"
```
Comment (dmanjunath) [inline .circleci/config.yml]: now that we have the landing site public maybe we can use that? 🙂

File: .circleci/config.yml [hunk 2]
```diff
@@ -162,19 +162,81 @@ jobs:
                     rmdir "$extracted_folder"
                   fi
 
+      - run:
+          name: Check for index.html
+          command: |
+            cd ~/project/repository
+            OUTPUT_DIR="."
```
Comment (dmanjunath) [inline .circleci/config.yml:169]: not sure i understand why this logic seems so complex. is it not possible to have something like?

if OUTPUT_DIR != "" then
  cd OUTPUT_DIR
end

ls -la
if [ ! -f "index.html" ]; then
  ..
end

**Review by dmanjunath — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -162,19 +162,81 @@ jobs:
...
+              echo "Staying in current directory ($(pwd))"
+            fi
+
+            echo "Current working directory:"
+            pwd
+
+            echo "Listing current directory contents:"
+            ls -la
+
+            bundle_filename="$(npx --package=armada-cli@0.4.1 --yes armada bundle create armada-bundle .)"
```
Comment (dmanjunath) [inline .circleci/config.yml]: we should also use the latest earthfast-cli@0.6.0 package

File: .circleci/config.yml (hunk 1 above)
Comment (jlmonroy13) [reply .circleci/config.yml]: oh yep, I forgot that, sorry

File: .circleci/config.yml (hunk 2 above)
Comment (jlmonroy13) [reply .circleci/config.yml:169]: It is to make the test work with the default values, only want to use the OUTPUT_DIR default value when the repo is the default one

Comment (dmanjunath) [reply .circleci/config.yml:169]: when you say test is that when a new commit is pushed to the runner repo it fails without  the default values?

File: .circleci/config.yml [hunk 4]
```diff
@@ -163,27 +163,89 @@ jobs:
...
+      - run:
+          name: Create earthfast release
           working_directory: ~/project/repository
           command: |
-            OUTPUT_DIR=.
-            if [ -n "<< pipeline.parameters.OUTPUT_DIR >>" ]; then
-              echo "Changing to output directory: << pipeline.parameters.OUTPUT_DIR >>"
-              OUTPUT_DIR="<< pipeline.parameters.OUTPUT_DIR >>"
+            # Check if we're using the default repository
+            if [ "<< pipeline.parameters.REPOSITORY >>" == "https://github.com/jlmonroy13/vite-project" ]; then
```
Comment (cheran-senthil) [inline .circleci/config.yml:212]: need to update this

**Pull request closed without merging**